src/pages/Appointments/utils.ts

```diff
@@ -79,7 +79,9 @@ export const useAvailabilityHeatmap = ({
   let queryFn = query(scheduleApis.slots.availabilityStats, {
     pathParams: { facility_id: facilityId },
     body: {
-      user: userId,
+      // voluntarily coalesce to empty string since we know query would be
+      // enabled only if userId is present
+      user: userId ?? "",
```

Comment:
Don't we need to add enabled for this query? 🤔

Reply:
it's there :) 


![image](https://github.com/user-attachments/assets/585cf5ad-5ee2-48a1-9d3e-2396ab25d6ab)